src/components/primitives/Select/Select.js

```diff
@@ -1,48 +1,64 @@
+import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { getDownArrowIcon } from './icons';
+import { css } from '@styled-system/css';
+import Flex from '../Flex';
+import IconBase from '../Icon';
+
+const Icon = styled(IconBase)({
```

Comment:
I would instead use `css` prop in the render methods on the component

Reply:
Looks like it doesn't work in Styleguidist or there's a problem somewhere else when I import from `*/macros`. I had the same error about `fs` that we've seen on the workshop ;-/

Reply:
Can you file an issue about it so I can take a look?

Reply:
https://github.com/component-driven/component-driven-development/issues/60